mne/viz/_figure.py

```diff
@@ -695,6 +695,7 @@ def _create_ch_location_fig(self, idx):
         # fig.lasso.alpha_other = 0.3
         # fig.lasso.linewidth_selected = 3
         # fig.lasso.style_sensors(inds)
+        # plt_show(fig=fig)
```

Comment:
cleanup?

Reply:
I think these lines were left here on purpose…

Reply:
indeed, those are dealt with in #8381